sonar-flex-plugin/src/main/java/org/sonar/plugins/flex/cobertura/CoberturaReportParser.java

```diff
@@ -125,4 +135,49 @@ private static void collectFileData(SMInputCursor clazz, NewCoverage newCoverage
     }
     newCoverage.save();
   }
+
+  /**
+   * StaxParser copied and simplified from deprecated StaxParser SQ API 5.6
+   */
+  private static class StaxParser {
+
+    private XmlStreamHandler streamHandler;
+    private SMInputFactory inf;
+
+    public StaxParser(XmlStreamHandler streamHandler) {
+      this.streamHandler = streamHandler;
+      XMLInputFactory xmlFactory = XMLInputFactory.newInstance();
+      if (xmlFactory instanceof WstxInputFactory) {
+        WstxInputFactory wstxInputfactory = (WstxInputFactory) xmlFactory;
+        wstxInputfactory.configureForLowMemUsage();
+        wstxInputfactory.getConfig().setUndeclaredEntityResolver((publicID, systemID, baseURI, namespace) -> namespace);
```

Comment:
Is that equivalent to what SQ's StaxParser does?

Reply:
Not really, but that's how we approximated it in Java. I was not sure about keeping the current logic from SQ, as no tests were failing.